bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
```

Comment:
I think it's purely written in Go. The `tomlv` is an Go executable program that run the main TOML validator library.